src/main/scala/com/fulcrumgenomics/umi/CollectDuplexSeqMetrics.scala

```diff
@@ -0,0 +1,501 @@
+    * example if a tag family is comprised of three read pairs with UMIs ACGT-TGGT ACGT-TGGT ACGT-TGGG that
+    * a consensus UMI of will be generated ACGT-TGGT will be generated, and 3 raw observations counted for each
+    * of ACGT and TGGT, and no observations counted for TGGG.
+    *
+    * @param umi the possibly-corrected UMI sequence
+    * @param raw_observations the number of read pairs in the input BAM that observe the UMI (after correction)
+    * @param raw_observations_with_errors the subset of raw-observations that underwent any correction
+    * @param unique_observations the number of double-stranded tag families (i.e unique double-stranded molecules)
+    *                            that observed the UMI
+    * @param fraction_raw_observations the fraction of all raw observations that the UMI accounts for
+    * @param fraction_unique_observations the fraction of all unique observations that the UMI accounts for
+    */
+  case class UmiMetric(umi: String,
+                       var raw_observations: Long = 0,
+                       var raw_observations_with_errors: Long = 0,
+                       var unique_observations: Long = 0,
+                       var fraction_raw_observations: Double = 0,
+                       var fraction_unique_observations: Double = 0
+                      ) extends Metric
+}
+
+// TODOs:
+// TODO: Better minor breaks for log-axis plots
+
+
+@clp(group=ClpGroups.Umi, description=
+  """
+    |Collects a suite of metrics to QC duplex sequencing data.
+    |
+    |Usage to be finalized when tool is finalized, but to include:
+    |  - Emphasis tha the input must be the BAM produced by GroupReadsByUmi with strategy=paired
+    |  - Definition of CS, SS and DS families and how they relate
+    |  - Definition of "duplex" and how minAbReads and minBaReads relate to it
+    |  - 1-2 sentence summary for each the various metrics files produced
+    |
+    |For plots to be generated R must be installed and the ggplot2 package installed with suggested
+    |dependencies. Successfully executing the following in R will ensure a working installation:
+    |    install.packages("ggplot2", repos="http://cran.us.r-project.org", dependencies=TRUE)
+  """)
+class CollectDuplexSeqMetrics
+( @arg(flag="i", doc="Input BAM file generated by GroupReadByUmi.") val input: PathToBam,
+  @arg(flag="o", doc="Prefix of output files to write.") val output: PathPrefix,
+  @arg(flag="l", doc="Optional set of intervals over which to restrict analysis.") val intervals: Option[PathToIntervals] = None,
+  @arg(flag="d", doc="Description of dataset used to label plots. Defaults to sample/library.") val description: Option[String] = None,
+  @arg(flag="a", doc="Minimum AB reads to call a tag family a 'duplex'.") val minAbReads: Int = 1,
+  @arg(flag="b", doc="Minimum BA reads to call a tag family a 'duplex'.") val minBaReads: Int = 1
+) extends FgBioTool with LazyLogging {
+  import CollectDuplexSeqMetrics._
```

Comment:
Yes, but it can't be expressed as a sort order.  It requires things to be in the order that they are produced in by GroupReadsByUmi.  Hence the doc for input = `Input BAM file generated by GroupReadByUmi`.  That order is super useful, but it's not a `SortOrder` - it's sorted by the lower 5' coordinate of the pair, and then by MI.